### PR TITLE
Replace tableInspector with infoGetter

### PR DESCRIPTION
**Pull request by suzaku**
### What problem does this PR solve? 

The information we need from tableInspector can be queried from infoGetter, so we can just replace it with infoGetter.

### What is changed and how it works?

Replace tableInspector with infoGetter, remove related implementation.

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: cdc/schema/storage.go [hunk 1]
```diff
@@ -103,6 +103,41 @@ func (ti *TableInfo) GetIndexInfo(indexID int64) (info *model.IndexInfo, exist b
...
+	}
+	return cols
+}
+
+// GetUniqueKeys returns all unique keys of the table as a slice of column names
+func (ti *TableInfo) GetUniqueKeys() [][]string {
+	var uniqueKeys [][]string
+	for _, idx := range ti.Indices {
+		if idx.Primary || idx.Unique {
+			colNames := make([]string, 0, len(idx.Columns))
```
Comment (leoppro) [inline cdc/schema/storage.go]: we should filter nullable column here

File: cdc/schema/storage.go [hunk 2]
```diff
@@ -103,6 +103,41 @@ func (ti *TableInfo) GetIndexInfo(indexID int64) (info *model.IndexInfo, exist b
...
+	}
+	if ti.PKIsHandle {
+		for _, col := range ti.Columns {
+			if mysql.HasPriKeyFlag(col.Flag) {
+				// Prepend to make sure the primary key ends up at the front
+				uniqueKeys = append([][]string{{col.Name.O}}, uniqueKeys...)
+				break
+			}
+		}
+	}
```
Comment (leoppro) [inline cdc/schema/storage.go]: should we make sure that the `pk` is the first in unique keys?

Comment (suzaku) [reply cdc/schema/storage.go]: I'm not sure, just to make it compatible with the original `inspector` implementation.

**Comment by leoppro** [comment]
LGTM

**Pull request merged**